### PR TITLE
[Ripple] Card with Ripple example now allows the card to be reached with voiceover.

### LINKED ISSUE #8914: [Catalog] Internal issue: b/144563624
This was filed as an internal issue. If you are a Googler, please visit [b/144563624](http://b/144563624) for more details.

---
#### Internal data
- Associated internal bug: [b/144563624](http://b/144563624)

### PR DESCRIPTION
**Pull request by yarneo**
Voice Over and Switch Control users can activate the Card to show Ripple.
The card has an accessibility label and an accessibility trait to indicate that it is interactive.

Closes #8914

### CODE REVIEW

File: components/Ripple/examples/CardWithRippleExample.swift [hunk 1]
```diff
@@ -64,6 +64,11 @@ class CardWithRippleExample: UIViewController {
     button.addConstraint(NSLayoutConstraint(item: button, attribute: .height, relatedBy: .equal,
                                             toItem: nil, attribute: .notAnAttribute, multiplier: 1,
                                             constant: 60))
+
+    view.accessibilityElements = [card, button]
```
Comment (jverkoey) [inline components/Ripple/examples/CardWithRippleExample.swift:68]: Why is this line necessary?

**Review by jverkoey — APPROVED** [review]

File: components/Ripple/examples/CardWithRippleExample.swift (hunk 1 above)
Comment (yarneo) [reply components/Ripple/examples/CardWithRippleExample.swift:68]: yeah, otherwise the button which is a subview of the card would not be accessible
